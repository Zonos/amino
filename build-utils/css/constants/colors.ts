export const colors = {
  black: '#0c0c0d',
  /* GRAY PALETTE */
  'gray-l80': '#f5f5f6',
  'gray-l60': '#eaeaec',
  'gray-l40': '#d6d6d9',
  'gray-l20': '#b7b7bd',
  'gray-base': '#9898a0',
  'gray-d20': '#7a7a80',
  'gray-d40': '#5b5b60',
  'gray-d60': '#3d3d40',
  'gray-d80': '#232326',
  'gray-alpha': '#f9fafb',
  'gray-50': '#f9fafb',
  'gray-100': 'var(--amino-gray-l80)',
  'gray-200': 'var(--amino-gray-l60)',
  'gray-300': 'var(--amino-gray-l40)',
  'gray-400': 'var(--amino-gray-l20)',
  'gray-500': 'var(--amino-gray-base)',
  'gray-600': 'var(--amino-gray-d20)',
  'gray-700': 'var(--amino-gray-d40)',
  'gray-800': 'var(--amino-gray-d60)',
  'gray-900': 'var(--amino-gray-d80)',
  /* BLUE PALETTE */
  'blue-l80': '#e9ebff',
  'blue-l60': '#a7afff',
  'blue-l40': '#7b86ff',
  'blue-l20': '#4f5eff',
  'blue-base': '#2337ff',
  'blue-d20': '#1c2aca',
  'blue-d40': '#152099',
  'blue-d60': '#0f1666',
  'blue-d80': '#1d233b',
  'blue-alpha': '#2190ff80',
  'blue-50': '#ebf5ff',
  'blue-100': 'var(--amino-blue-l80)',
  'blue-200': 'var(--amino-blue-l60)',
  'blue-300': 'var(--amino-blue-l40)',
  'blue-400': 'var(--amino-blue-l20)',
  'blue-500': 'var(--amino-blue-base)',
  'blue-600': 'var(--amino-blue-d20)',
  'blue-700': 'var(--amino-blue-d40)',
  'blue-800': 'var(--amino-blue-d60)',
  'blue-900': 'var(--amino-blue-d80)',
  /* CYAN PALETTE */
  'cyan-l80': '#e9fcff',
  'cyan-l60': '#a5edf6',
  'cyan-l40': '#77e3f2',
  'cyan-l20': '#4adaed',
  'cyan-base': '#1dd1e9',
  'cyan-d20': '#17a7ba',
  'cyan-d40': '#117d8c',
  'cyan-d60': '#0c545d',
  'cyan-d80': '#1d363b',
  'cyan-alpha': '#00dfe380',
  'cyan-50': '#e9fcff',
  'cyan-100': 'var(--amino-cyan-l80)',
  'cyan-200': 'var(--amino-cyan-l60)',
  'cyan-300': 'var(--amino-cyan-l40)',
  'cyan-400': 'var(--amino-cyan-l20)',
  'cyan-500': 'var(--amino-cyan-base)',
  'cyan-600': 'var(--amino-cyan-d20)',
  'cyan-700': 'var(--amino-cyan-d40)',
  'cyan-800': 'var(--amino-cyan-d60)',
  'cyan-900': 'var(--amino-cyan-d80)',
  /* RED PALETTE */
  'red-l80': '#fde5eb',
  'red-l60': '#f899b1',
  'red-l40': '#f5668a',
  'red-l20': '#f13363',
  'red-base': '#ee003c',
  'red-d20': '#be0030',
  'red-d40': '#8f0024',
  'red-d60': '#5f0018',
  'red-d80': '#371822',
  'red-alpha': '#e0451a80',
  'red-50': '#fef2f2',
  'red-100': 'var(--amino-red-l80)',
  'red-200': 'var(--amino-red-l60)',
  'red-300': 'var(--amino-red-l40)',
  'red-400': 'var(--amino-red-l20)',
  'red-500': 'var(--amino-red-base)',
  'red-600': 'var(--amino-red-d20)',
  'red-700': 'var(--amino-red-d40)',
  'red-800': 'var(--amino-red-d60)',
  'red-900': 'var(--amino-red-d80)',
  /* ORANGE PALETTE */
  'orange-l80': '#fff0e5',
  'orange-l60': '#ffc499',
  'orange-l40': '#ffa666',
  'orange-l20': '#ff8933',
  'orange-base': '#ff6b00',
  'orange-d20': '#cc5600',
  'orange-d40': '#994000',
  'orange-d60': '#662b00',
  'orange-d80': '#39271c',
  'orange-alpha': '#ffa51e80',
  'orange-50': '#fff0e5',
  'orange-100': 'var(--amino-orange-l80)',
  'orange-200': 'var(--amino-orange-l60)',
  'orange-300': 'var(--amino-orange-l40)',
  'orange-400': 'var(--amino-orange-l20)',
  'orange-500': 'var(--amino-orange-base)',
  'orange-600': 'var(--amino-orange-d20)',
  'orange-700': 'var(--amino-orange-d40)',
  'orange-800': 'var(--amino-orange-d60)',
  'orange-900': 'var(--amino-orange-d80)',
  /* GREEN PALETTE */
  'green-l80': '#e5f9e5',
  'green-l60': '#99e799',
  'green-l40': '#66db66',
  'green-l20': '#33cf33',
  'green-base': '#00c300',
  'green-d20': '#009c00',
  'green-d40': '#007500',
  'green-d60': '#004e00',
  'green-d80': '#17311a',
  'green-alpha': '#ecfdf580',
  'green-50': '#ecfdf5',
  'green-100': 'var(--amino-green-l80)',
  'green-200': 'var(--amino-green-l60)',
  'green-300': 'var(--amino-green-l40)',
  'green-400': 'var(--amino-green-l20)',
  'green-500': 'var(--amino-green-base)',
  'green-600': 'var(--amino-green-d20)',
  'green-700': 'var(--amino-green-d40)',
  'green-800': 'var(--amino-green-d60)',
  'green-900': 'var(--amino-green-d80)',
  /* PURPLE */
  'purple-l80': '#f5e5ff',
  'purple-l60': '#d599ff',
  'purple-l40': '#c066ff',
  'purple-l20': '#ab33ff',
  'purple-base': '#9600ff',
  'purple-d20': '#7800cc',
  'purple-d40': '#5a0099',
  'purple-d60': '#3c0066',
  'purple-d80': '#2c183b',
  'purple-alpha': '#f5f3ff80',
  'purple-50': '#f5f3ff',
  'purple-100': 'var(--amino-purple-l80)',
  'purple-200': 'var(--amino-purple-l60)',
  'purple-300': 'var(--amino-purple-l40)',
  'purple-400': 'var(--amino-purple-l20)',
  'purple-500': 'var(--amino-purple-base)',
  'purple-600': 'var(--amino-purple-d20)',
  'purple-700': 'var(--amino-purple-d40)',
  'purple-800': 'var(--amino-purple-d60)',
  'purple-900': 'var(--amino-purple-d80)',
  /* YELLOW PALLETE */
  'yellow-l80': '#fff9e5',
  'yellow-l60': '#ffe999',
  'yellow-l40': '#ffdd66',
  'yellow-l20': '#ffd233',
  'yellow-base': '#ffc700',
  'yellow-d20': '#cc9f00',
  'yellow-d40': '#997700',
  'yellow-d60': '#665000',
  'yellow-d80': '#39321c',
  'yellow-alpha': '#fffbeb80',
  'yellow-50': '#fffbeb',
  'yellow-100': 'var(--amino-yellow-l80)',
  'yellow-200': 'var(--amino-yellow-l60)',
  'yellow-300': 'var(--amino-yellow-l40)',
  'yellow-400': 'var(--amino-yellow-l20)',
  'yellow-500': 'var(--amino-yellow-base)',
  'yellow-600': 'var(--amino-yellow-d20)',
  'yellow-700': 'var(--amino-yellow-d40)',
  'yellow-800': 'var(--amino-yellow-d60)',
  'yellow-900': 'var(--amino-yellow-d80)',
  'amber-alpha': '#fffbeb80',
  'amber-50': '#fffbeb',
  'amber-100': 'var(--amino-yellow-l80)',
  'amber-200': 'var(--amino-yellow-l60)',
  'amber-300': 'var(--amino-yellow-l40)',
  'amber-400': 'var(--amino-yellow-l20)',
  'amber-500': 'var(--amino-yellow-base)',
  'amber-600': 'var(--amino-yellow-d20)',
  'amber-700': 'var(--amino-yellow-d40)',
  'amber-800': 'var(--amino-yellow-d60)',
  'amber-900': 'var(--amino-yellow-d80)',
} as const;

export const commonColors = {
  danger: 'var(--amino-red-500)',
  'danger-dark': 'var(--amino-red-600)',
  error: 'var(--amino-red-500)',
  primary: 'var(--amino-blue-500)',
  'primary-light': 'var(--amino-blue-400)',
  'primary-dark': 'var(--amino-blue-700)',
  success: 'var(--amino-green-200)',
  'success-dark': 'var(--amino-green-600)',
  warning: 'var(--amino-amber-100)',
  'warning-dark': 'var(--amino-amber-600)',
} as const;

export const glowColors = {
  'glow-error': '0 0 0 1px var(--amino-red-500)',
  'glow-blue': '0 0 0 1px var(--amino-primary)',
  'glow-cyan': '0 0 0 1px var(--amino-cyan-alpha)',
  'glow-red': '0 0 0 1px var(--amino-red-alpha)',
  'glow-orange': '0 0 0 1px var(--amino-orange-alpha)',
  'glow-green': '0 0 0 1px var(--amino-green-alpha)',
  'glow-purple': '0 0 0 1px var(--amino-purple-alpha)',
} as const;

export const layoutColors = {
  'page-background': 'white',
  'border-color': 'var(--amino-gray-200)',
  'hover-color': 'var(--amino-gray-50)',
  'surface-color': 'white',
  'surface-color-secondary': 'var(--amino-gray-50)',
  'text-light': 'white',
  'text-color': 'var(--amino-gray-900)',
  'text-dark': 'var(--amino-text-color)',
  'input-background': 'white',
} as const;

export const colorStyleList = {
  ...colors,
  ...commonColors,
  ...glowColors,
  ...layoutColors,
};
