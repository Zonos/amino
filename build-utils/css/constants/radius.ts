export const radius = {
  /** @info 4px */
  'radius-sm': '0.25rem',
  /** @info 6px */
  radius: '0.375rem',
  /** @info 8px */
  'radius-lg': '0.5rem',
  /** @info 12px */
  'radius-xl': '0.75rem',
} as const;
