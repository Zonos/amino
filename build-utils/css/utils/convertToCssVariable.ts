export const convertToCssConstant = (themeContent: Record<string, string>) => {
  const newContent: Record<string, string> = {};
  Object.entries(themeContent).forEach(([key, value]) => {
    newContent[`--amino-${key}`] = value;
  });

  return newContent;
};
