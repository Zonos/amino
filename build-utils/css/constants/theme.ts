import { misc } from 'build-utils/css/constants/theme/_misc';
import { spacing } from 'build-utils/css/constants/theme/_spacing';
import { variablesUsingColors } from 'build-utils/css/constants/theme/_variablesUsingColors';
import { hover } from 'build-utils/css/constants/theme/day/_hover';
import { shadow } from 'build-utils/css/constants/theme/day/_shadow';
import { spinner } from 'build-utils/css/constants/theme/day/_spinner';
import { blue } from 'build-utils/css/constants/theme/day/colors/_blue';
import { cyan } from 'build-utils/css/constants/theme/day/colors/_cyan';
import { glass } from 'build-utils/css/constants/theme/day/colors/_glass';
import { gray } from 'build-utils/css/constants/theme/day/colors/_gray';
import { green } from 'build-utils/css/constants/theme/day/colors/_green';
import { oneOffColors } from 'build-utils/css/constants/theme/day/colors/_oneOffColors';
import { orange } from 'build-utils/css/constants/theme/day/colors/_orange';
import { purple } from 'build-utils/css/constants/theme/day/colors/_purple';
import { red } from 'build-utils/css/constants/theme/day/colors/_red';
import { transparent } from 'build-utils/css/constants/theme/day/colors/_transparent';

export const theme = {
  transition: 'all 0.2s ease-in-out 0s, visibility 0s',
  'type-scale-base': '16px',

  /* GLASS PALETTE */
  ...glass,

  /* GRAY PALETTE */
  ...gray,

  /* BLUE PALETTE */
  ...blue,

  /* CYAN PALETTE */
  ...cyan,

  /* RED PALETTE */
  ...red,

  /* ORANGE PALETTE */
  ...orange,

  /* GREEN PALETTE */
  ...green,

  /* PURPLE */
  ...purple,

  /* TRANSPARENT */
  ...transparent,

  /* BOX SHADOW */
  ...shadow,

  /* MISC */
  ...misc,

  /* SPACING */
  ...spacing,

  /* VARIABLES USING COLORS */
  ...variablesUsingColors,

  /* ONE OFF COLORS */
  ...oneOffColors,

  /* BUTTON HOVER COLORS */
  ...hover,

  /* SPINNER COLORS */
  ...spinner,

  /* LAYOUT STYLE */
  'appbar-height': '55px',
  'sidebar-width': '232px',
  'elevation-0': '0',
  'elevation-100': '100',
  'elevation-200': '200',
  'elevation-300': '300',
  'elevation-400': '400',
  'elevation-500': '500',
  /** @info 400 */
  'dialog-elevation': 'var(--amino-elevation-400)',
  /** @info 500 */
  'toast-elevation': 'var(--amino-elevation-500)',
  /** @info 500 */
  'appbar-elevation': 'var(--amino-elevation-500)',

  /* BUTTON SIZE */
  'size-xl': '56px',
  'size-lg': '48px',
  'size-md': '40px',
  'size-sm': '32px',

  /* BORDER RADIUS */
  /** @info 4px */
  'radius-4': '0.25rem',
  /** @info 6px */
  'radius-6': '0.375rem',
  /** @info 8px */
  'radius-8': '0.5rem',
  /** @info 10px */
  'radius-10': '0.625rem',
  /** @info 12px */
  'radius-12': '0.75rem',

  /** @info 30px */
  'font-size-3xl': '1.875rem',
  /** @info 24px  */
  'font-size-2xl': '1.5rem',
  /** @info 20px */
  'font-size-xl': '1.25rem',
  /** @info 18px */
  'font-size-l': '1.125rem',
  /** @info 16px */
  'font-size-base-docs': '1rem',
  /** @info 13px */
  'font-size-base': '0.813rem',
  /** @info 12px */
  'font-size-s': '0.75rem',
  /** @info 11px */
  'font-size-xs': '0.688rem',

  /* LINE HEIGHT */
  /** @info 48px */
  'line-height-3xl': '3rem',
  /** @info 40px */
  'line-height-2xl': '2.5rem',
  /** @info 32px */
  'line-height-xl': '2rem',
  /** @info 24px */
  'line-height-l': '1.5rem',
  /** @info 24px */
  'line-height-base': '1.5rem',
  /** @info 16px */
  'line-height-s': '1rem',
  /** @info 12px */
  'line-height-xs': '0.75rem',

  /* FONTS */
  'font-sans': `'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif`,
  'font-mono': `Operator Mono, MonoLisa, Dank Mono, Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace`,
  'font-harmonized-codes': `'Fira Code', var(--amino-font-sans)`,

  /* OTHER */
  'input-background': 'inherit',
} as const;

export type ThemeKey = keyof typeof theme;

export type ThemeValue = (typeof theme)[ThemeKey];
