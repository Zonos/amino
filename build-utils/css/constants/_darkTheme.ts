import { colorStyleList } from './colors';
import { layoutStyleList } from './layout';
import { constraintDefinedAminoVar } from './utils';

const darkColorStyleList = constraintDefinedAminoVar(colorStyleList, {
  'gray-l80': '#f5f5f6',
  'gray-l60': '#d8d8db',
  'gray-l40': '#c4c4c9',
  'gray-l20': '#b1b1b7',
  'gray-base': '#9d9da5',
  'gray-d20': '#76767c',
  'gray-d40': '#4f4f52',
  'gray-d60': '#2f2f31',
  'gray-d80': '#232326',
  /* BLUE PALETTE */
  'blue-l80': '#e9ebff',
  'blue-l60': '#b4c5ff',
  'blue-l40': '#8fa7ff',
  'blue-l20': '#698aff',
  'blue-base': '#446dff',
  'blue-d20': '#3657cc',
  'blue-d40': '#294199',
  'blue-d60': '#1b2c66',
  'blue-d80': '#1d233b',
  /* CYAN PALETTE */
  'cyan-l80': '#e9fcff',
  'cyan-l60': '#b5f6ff',
  'cyan-l40': '#8ff2ff',
  'cyan-l20': '#6aedff',
  'cyan-base': '#45e9ff',
  'cyan-d20': '#37bacc',
  'cyan-d40': '#298c99',
  'cyan-d60': '#1c5d66',
  'cyan-d80': '#1d363b',
  /* RED PALETTE */
  'red-l80': '#fde5eb',
  'red-l60': '#f9a8bd',
  'red-l40': '#f77d9b',
  'red-l20': '#f4517a',
  'red-base': '#f12659',
  'red-d20': '#c11e47',
  'red-d40': '#911735',
  'red-d60': '#600f24',
  'red-d80': '#371822',
  /* ORANGE PALETTE */
  'orange-l80': '#fff0e5',
  'orange-l60': '#ffd0ad',
  'orange-l40': '#ffb885',
  'orange-l20': '#ffa05c',
  'orange-base': '#ff8933',
  'orange-d20': '#cc6e29',
  'orange-d40': '#99521f',
  'orange-d60': '#663714',
  'orange-d80': '#39271c',
  /* GREEN PALETTE */
  'green-l80': '#e5f9e5',
  'green-l60': '#a5eba8',
  'green-l40': '#78e07d',
  'green-l20': '#4bd651',
  'green-base': '#1ecc26',
  'green-d20': '#18a31e',
  'green-d40': '#127a17',
  'green-d60': '#0c520f',
  'green-d80': '#17311a',
  /* PURPLE */
  'purple-l80': '#f5e5ff',
  'purple-l60': '#dba8ff',
  'purple-l40': '#ca7dff',
  'purple-l20': '#b851ff',
  'purple-base': '#a626ff',
  'purple-d20': '#851ecc',
  'purple-d40': '#641799',
  'purple-d60': '#420f66',
  'purple-d80': '#2c183b',
  /* YELLOW PALLET */
  'yellow-l80': '#fff9e5',
  'yellow-l60': '#ffedad',
  'yellow-l40': '#ffe485',
  'yellow-l20': '#ffdb5c',
  'yellow-base': '#ffd233',
  'yellow-d20': '#cca829',
  'yellow-d40': '#997e1f',
  'yellow-d60': '#665414',
  'yellow-d80': '#39321c',

  'page-background': 'var(--amino-gray-50)',
  'surface-color': 'var(--amino-gray-500)',
  'surface-color-secondary': 'var(--amino-gray-50)',
  'border-color': 'var(--amino-gray-200)',
  'text-color': 'white',
  'input-background': 'var(--amino-gray-300)',
  'hover-color': 'var(--amino-gray-400)',
});

const darkLayoutStyleList = constraintDefinedAminoVar(layoutStyleList, {
  'v3-shadow-inset': 'inset 0px 2px 4px rgba(0, 0, 0, 0.18)',
  'v3-shadow-base':
    '0px 1px 2px rgba(0, 0, 0, 0.18), 0px 1px 3px rgba(0, 0, 0, 0.3)',
  'v3-shadow-medium':
    '0px 2px 4px rgba(0, 0, 0, 0.18), 0px 4px 6px rgba(0, 0, 0, 0.3)',
  'v3-shadow-large':
    '0px 10px 15px rgba(0, 0, 0, 0.3), 0px 4px 6px rgba(0, 0, 0, 0.15)',
  'v3-shadow-xl':
    '0px 20px 25px rgba(0, 0, 0, 0.3), 0px 10px 10px rgba(0, 0, 0, 0.12)',
  'v3-shadow-xxl': '0px 32px 50px rgba(0, 0, 0, 0.72)',
  'backdrop-color': 'var(--amino-gray-200)',
  'header-color': 'var(--amino-gray-100)',
  border: '1px solid var(--amino-border-color)',
});

export const darkStyleList = {
  ...darkColorStyleList,
  ...darkLayoutStyleList,
};
