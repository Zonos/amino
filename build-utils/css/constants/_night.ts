import { theme } from './theme';
import { blue } from './theme/night/colors/_blue';
import { cyan } from './theme/night/colors/_cyan';
import { gray } from './theme/night/colors/_gray';
import { red } from './theme/night/colors/_red';
import { orange } from './theme/night/colors/_orange';
import { green } from './theme/night/colors/_green';
import { purple } from './theme/night/colors/_purple';
import { transparent } from './theme/night/colors/_transparent';

import { constraintDefinedAminoVar } from './utils';

export const night = constraintDefinedAminoVar(theme, {
  /* GRAY PALETTE */
  ...gray,

  /* BLUE PALETTE */
  ...blue,

  /* CYAN PALETTE */
  ...cyan,

  /* RED PALETTE */
  ...red,

  /* ORANGE PALETTE */
  ...orange,

  /* GREEN PALETTE */
  ...green,

  /* PURPLE */
  ...purple,

  /* TRANSPARENT */
  ...transparent,

  'page-background': 'var(--amino-gray-50)',
  'surface-color': 'var(--amino-gray-base)',
  'surface-color-secondary': 'var(--amino-gray-50)',
  'border-color': 'var(--amino-gray-l60)',
  'text-color': 'white',
  'input-background': 'var(--amino-gray-l40)',
  'hover-color': 'var(--amino-gray-l20)',

  'v3-shadow-inset': 'inset 0px 2px 4px rgba(0, 0, 0, 0.18)',
  'v3-shadow-base':
    '0px 1px 2px rgba(0, 0, 0, 0.18), 0px 1px 3px rgba(0, 0, 0, 0.3)',
  'v3-shadow-medium':
    '0px 2px 4px rgba(0, 0, 0, 0.18), 0px 4px 6px rgba(0, 0, 0, 0.3)',
  'v3-shadow-large':
    '0px 10px 15px rgba(0, 0, 0, 0.3), 0px 4px 6px rgba(0, 0, 0, 0.15)',
  'v3-shadow-xl':
    '0px 20px 25px rgba(0, 0, 0, 0.3), 0px 10px 10px rgba(0, 0, 0, 0.12)',
  'v3-shadow-xxl': '0px 32px 50px rgba(0, 0, 0, 0.72)',
  'backdrop-color': 'var(--amino-gray-l60)',
  'header-color': 'var(--amino-gray-l80)',
  border: '1px solid var(--amino-border-color)',
});
