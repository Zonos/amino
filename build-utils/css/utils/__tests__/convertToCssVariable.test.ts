import { theme } from '../../constants/theme';
import { convertToCssConstant } from '../convertToCssVariable';

test(`Theme content:`, () =>
  expect(convertToCssConstant(theme)).toMatchSnapshot());
