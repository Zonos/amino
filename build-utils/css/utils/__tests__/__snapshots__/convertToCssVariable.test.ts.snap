// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme content: 1`] = `
Object {
  "--amino-appbar-elevation": "var(--amino-elevation-500)",
  "--amino-appbar-height": "55px",
  "--amino-backdrop-color": "var(--amino-gray-1000)",
  "--amino-black": "#0c0c0d",
  "--amino-blue-100": "#DEECFE",
  "--amino-blue-1000": "#151943",
  "--amino-blue-200": "#C5DCFD",
  "--amino-blue-300": "#AAC8FC",
  "--amino-blue-400": "#89ADFA",
  "--amino-blue-500": "#568CF6",
  "--amino-blue-600": "#435DEE",
  "--amino-blue-700": "#294DBE",
  "--amino-blue-800": "#263B98",
  "--amino-blue-900": "#1F2B73",
  "--amino-border": "1px solid var(--amino-border-color)",
  "--amino-border-amber": "1px solid var(--amino-amber-400)",
  "--amino-border-blue": "1px solid var(--amino-blue-500)",
  "--amino-border-color": "var(--amino-gray-200)",
  "--amino-border-cyan": "1px solid var(--amino-cyan-400)",
  "--amino-border-green": "1px solid var(--amino-green-400)",
  "--amino-border-orange": "1px solid var(--amino-orange-400)",
  "--amino-border-purple": "1px solid var(--amino-purple-400)",
  "--amino-border-red": "1px solid var(--amino-red-400)",
  "--amino-border-transparent": "1px solid transparent",
  "--amino-cyan-100": "#DBFDFF",
  "--amino-cyan-1000": "#183338",
  "--amino-cyan-200": "#BAF8FD",
  "--amino-cyan-300": "#8FF1FA",
  "--amino-cyan-400": "#65E2EF",
  "--amino-cyan-500": "#48C4D3",
  "--amino-cyan-600": "#349DAB",
  "--amino-cyan-700": "#287E8A",
  "--amino-cyan-800": "#236A75",
  "--amino-cyan-900": "#22525A",
  "--amino-danger": "var(--amino-red-600)",
  "--amino-danger-dark": "var(--amino-red-700)",
  "--amino-dialog-elevation": "var(--amino-elevation-400)",
  "--amino-elevation-0": "0",
  "--amino-elevation-100": "100",
  "--amino-elevation-200": "200",
  "--amino-elevation-300": "300",
  "--amino-elevation-400": "400",
  "--amino-elevation-500": "500",
  "--amino-error": "var(--amino-red-600)",
  "--amino-font-harmonized-codes": "'Fira Code', var(--amino-font-sans)",
  "--amino-font-mono": "Operator Mono, MonoLisa, Dank Mono, Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace",
  "--amino-font-sans": "'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif",
  "--amino-font-size-2xl": "1.75rem",
  "--amino-font-size-3xl": "2.125rem",
  "--amino-font-size-base": "0.875rem",
  "--amino-font-size-l": "1.125rem",
  "--amino-font-size-s": "0.75rem",
  "--amino-font-size-xl": "1.375rem",
  "--amino-font-size-xs": "0.625rem",
  "--amino-glow-blue": "0 0 0 1px var(--amino-primary)",
  "--amino-glow-cyan": "0 0 0 1px var(--amino-cyan-alpha)",
  "--amino-glow-error": "0 0 0 1px var(--amino-red-600)",
  "--amino-glow-green": "0 0 0 1px var(--amino-green-alpha)",
  "--amino-glow-orange": "0 0 0 1px var(--amino-orange-alpha)",
  "--amino-glow-purple": "0 0 0 1px var(--amino-purple-alpha)",
  "--amino-glow-red": "0 0 0 1px var(--amino-red-alpha)",
  "--amino-gray-0": "white",
  "--amino-gray-100": "#EAEBED",
  "--amino-gray-1000": "#2E3038",
  "--amino-gray-1100": "#1E1F26",
  "--amino-gray-1200": "#101116",
  "--amino-gray-1300": "black",
  "--amino-gray-200": "#D9DADD",
  "--amino-gray-300": "#C5C6CA",
  "--amino-gray-400": "#ACAEB3",
  "--amino-gray-50": "#F6F6F8",
  "--amino-gray-500": "#94969E",
  "--amino-gray-600": "#6A6D77",
  "--amino-gray-700": "#5C5E68",
  "--amino-gray-800": "#4D505A",
  "--amino-gray-900": "#40424C",
  "--amino-green-100": "#DBFFDD",
  "--amino-green-1000": "#193123",
  "--amino-green-200": "#BAFDC2",
  "--amino-green-300": "#86F99E",
  "--amino-green-400": "#62E488",
  "--amino-green-500": "#4DC477",
  "--amino-green-600": "#2A995A",
  "--amino-green-700": "#25794A",
  "--amino-green-800": "#236641",
  "--amino-green-900": "#214C34",
  "--amino-header-color": "white",
  "--amino-hover-color": "var(--amino-gray-50)",
  "--amino-input-background": "white",
  "--amino-line-height-2xl": "119%",
  "--amino-line-height-3xl": "118%",
  "--amino-line-height-base": "114%",
  "--amino-line-height-l": "133%",
  "--amino-line-height-s": "133%",
  "--amino-line-height-xl": "127%",
  "--amino-line-height-xs": "120%",
  "--amino-orange-100": "#FFEEE0",
  "--amino-orange-1000": "#3C1611",
  "--amino-orange-200": "#FED5B8",
  "--amino-orange-300": "#FDC2A0",
  "--amino-orange-400": "#FCA375",
  "--amino-orange-500": "#F88548",
  "--amino-orange-600": "#D7581F",
  "--amino-orange-700": "#A4431D",
  "--amino-orange-800": "#85341D",
  "--amino-orange-900": "#63261A",
  "--amino-page-background": "white",
  "--amino-primary": "var(--amino-blue-600)",
  "--amino-primary-dark": "var(--amino-blue-800)",
  "--amino-primary-light": "var(--amino-blue-500)",
  "--amino-purple-100": "#EEF0FF",
  "--amino-purple-1000": "#29143A",
  "--amino-purple-200": "#D8DAFD",
  "--amino-purple-300": "#C4C0FC",
  "--amino-purple-400": "#B09FF9",
  "--amino-purple-500": "#A379F6",
  "--amino-purple-600": "#9635EE",
  "--amino-purple-700": "#7728B9",
  "--amino-purple-800": "#612392",
  "--amino-purple-900": "#461F66",
  "--amino-radius-10": "0.625rem",
  "--amino-radius-12": "0.75rem",
  "--amino-radius-4": "0.25rem",
  "--amino-radius-6": "0.375rem",
  "--amino-radius-8": "0.5rem",
  "--amino-red-100": "#FFEDEB",
  "--amino-red-1000": "#3B1119",
  "--amino-red-200": "#FDD3D0",
  "--amino-red-300": "#FCB4B2",
  "--amino-red-400": "#FA8B8B",
  "--amino-red-500": "#F9505B",
  "--amino-red-600": "#D3203F",
  "--amino-red-700": "#A81E37",
  "--amino-red-800": "#841A2F",
  "--amino-red-900": "#601725",
  "--amino-sidebar-color": "white",
  "--amino-sidebar-width": "264px",
  "--amino-size-lg": "48px",
  "--amino-size-md": "40px",
  "--amino-size-sm": "32px",
  "--amino-size-xl": "56px",
  "--amino-space-0": "0",
  "--amino-space-12": "0.75rem",
  "--amino-space-16": "1rem",
  "--amino-space-20": "1.25rem",
  "--amino-space-24": "1.5rem",
  "--amino-space-28": "1.75rem",
  "--amino-space-32": "2rem",
  "--amino-space-36": "2.25rem",
  "--amino-space-4": "0.25rem",
  "--amino-space-40": "2.5rem",
  "--amino-space-44": "2.75rem",
  "--amino-space-48": "3rem",
  "--amino-space-52": "3.25rem",
  "--amino-space-56": "3.5rem",
  "--amino-space-60": "3.75rem",
  "--amino-space-64": "4rem",
  "--amino-space-68": "4.25rem",
  "--amino-space-72": "4.5rem",
  "--amino-space-76": "4.75rem",
  "--amino-space-8": "0.5rem",
  "--amino-space-80": "5rem",
  "--amino-space-negative-12": "-0.75rem",
  "--amino-space-negative-16": "-1rem",
  "--amino-space-negative-20": "-1.25rem",
  "--amino-space-negative-24": "-1.5rem",
  "--amino-space-negative-28": "-1.75rem",
  "--amino-space-negative-32": "-2rem",
  "--amino-space-negative-36": "-2.25rem",
  "--amino-space-negative-4": "-0.25rem",
  "--amino-space-negative-40": "-2.5rem",
  "--amino-space-negative-44": "-2.75rem",
  "--amino-space-negative-48": "-3rem",
  "--amino-space-negative-52": "-3.25rem",
  "--amino-space-negative-56": "-3.5rem",
  "--amino-space-negative-60": "-3.75rem",
  "--amino-space-negative-64": "-4rem",
  "--amino-space-negative-68": "-4.25rem",
  "--amino-space-negative-72": "-4.5rem",
  "--amino-space-negative-76": "-4.75rem",
  "--amino-space-negative-8": "-0.5rem",
  "--amino-space-negative-80": "-5rem",
  "--amino-success": "var(--amino-green-300)",
  "--amino-success-dark": "var(--amino-green-700)",
  "--amino-surface-color": "white",
  "--amino-surface-color-secondary": "var(--amino-gray-50)",
  "--amino-text-color": "var(--amino-gray-1000)",
  "--amino-text-dark": "var(--amino-text-color)",
  "--amino-text-light": "white",
  "--amino-toast-elevation": "var(--amino-elevation-500)",
  "--amino-transition": "all 0.2s ease-in-out 0s",
  "--amino-type-scale-base": "16px",
  "--amino-v3-shadow-base": "0px 1px 2px rgba(0, 0, 0, 0.06), 0px 1px 3px rgba(0, 0, 0, 0.1)",
  "--amino-v3-shadow-inset": "inset 0px 2px 4px rgba(0, 0, 0, 0.06)",
  "--amino-v3-shadow-large": "0px 10px 15px rgba(0, 0, 0, 0.1), 0px 4px 6px rgba(0, 0, 0, 0.05)",
  "--amino-v3-shadow-medium": "0px 2px 4px rgba(0, 0, 0, 0.06), 0px 4px 6px rgba(0, 0, 0, 0.1)",
  "--amino-v3-shadow-xl": "0px 20px 25px rgba(0, 0, 0, 0.1), 0px 10px 10px rgba(0, 0, 0, 0.04)",
  "--amino-v3-shadow-xxl": "0px 32px 50px rgba(0, 0, 0, 0.24)",
  "--amino-warning": "var(--amino-amber-100)",
  "--amino-warning-dark": "var(--amino-amber-700)",
}
`;
